docs/source/variables.ipynb

```diff
@@ -104,8 +104,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## TextT\n",
-    "Text is a long-form strings, that can be of any length. It is commonly used with NLP operations, such as TF-IDF. Featuretools supports NLP operations with nlp-primitives [add-on](https://innovation.alteryx.com/natural-language-processing-featuretools/). Text variable type can be represented as strings. "
+    "## Text\n",
+    "Text is a long-form strings, that can be of any length. It is commonly used with NLP operations, such as TF-IDF. Featuretools supports NLP operations with nlp-primitives [add-on](https://innovation.alteryx.com/natural-language-processing-featuretools/)."
```

Comment:
> Text is a long-form strings

maybe "long-form string"

> Featuretools supports NLP operations with nlp-primitives add-on

perhaps "with the nlp-primitives add-on"

Reply:
Fixed